content/applications/hr/lunch/management.rst

```diff
@@ -0,0 +1,187 @@
+
+   The manager first marks the three pizzas as received, by individually confirming the products
+   with the :guilabel:`✔️ Confirm` icon at the end of each product line.
+
+   Later, when the vendor delivers the garlic knots, the manager can either click the :guilabel:`✔️
+   Confirm` icon at the end of the line for the garlic knots, or click the :guilabel:`Confirm
+   Orders` button that appears next to the vendor's name and phone number.
+
+.. _lunch/notify:
+
+Notify employees
+----------------
+
+After products are received, and the orders are confirmed, the employees **must** be informed that
+their orders have been delivered, and are ready to be picked up.
+
+Unlike sending and confirming orders, notifications must be sent individually, and cannot be sent in
+a batch.
+
+To notify the user their product has arrived, click the :guilabel:`✉️ Send Notification` icon at the
+end of each product line. An email is sent to the user informing them their products have been
+delivered.
+
+.. _lunch/control_vendors:
+
+Control Vendors
+===============
+
+All orders for all vendors, both past and present, can be found in the *Control Vendors* dashboard.
+To access these records, navigate to :menuselection:`Lunch app --> Manager --> Control Vendors`.
+
+All orders appear in a list view, grouped alphabetically by :guilabel:`Vendor`. The list loads with
+all vendors expanded to show all order lines for every vendor, by default.
+
+The following information appears in the list:
+
+- :guilabel:`Order Date`: the date the order was placed.
+- :guilabel:`Vendor`: the vendor the product is being ordered from.
+- :guilabel:`Product`: the specific product ordered.
+- :guilabel:`Extras`: any extras selected for the product.
+- :guilabel:`Notes`: any information needed to be sent to the vendor.
+- :guilabel:`User`: the user who ordered the product.
+- :guilabel:`Lunch Location`: where the product is set to be delivered.
+- :guilabel:`Price`: the total price for the product, including all extras.
+- :guilabel:`Status`: the current status of the product.
+- :guilabel:`Company`: the company under which the order was placed. This only appears in a
+  multi-company database.
+
+Orders can be :ref:`canceled <lunch/cancel>`, :ref:`sent to the vendor <lunch/send-orders>`,
+:ref:`confirmed <lunch/confirm-orders>` upon arrival, and :ref:`employees can be notified
+<lunch/notify>` using the same method as on the :ref:`Today's Orders <lunch/todays-orders>`
+dashboard.
+
+.. image:: management/control.png
+   :align: center
+   :alt: A list view of all the orders as seen in the Control Vendors dashboard.
+
+.. note::
+   The difference between the :ref:`Today's Orders <lunch/todays-orders>` dashboard and the
+   :ref:`Control Vendors <lunch/control_vendors>` dashboard is that the *Today's Orders* dashboard
+   **only** displays orders for the current day, while the :guilabel:`Control Vendors` dashboard
+   displays **all** orders made in the *Lunch* app.
```

Comment:
It may be helpful to link the other docs?
```suggestion
   displays **all** orders made in the *Lunch* app.
   
.. seealso::
   - :doc:`../lunch`
   - :doc:`orders`
   - :doc:`user-accounts`
```